src/template/option.json

```diff
@@ -1,6 +1,7 @@
 {
-    "id": "",
-    "type": "string",
     "description": "",
+    "type": "string",
+    "named": null,
+    "usage": "",
```

Comment:
I'm not sure why do we need this, especially when it's not correct (named and usage props), and optional props are generated with default values. I recommend to skip it.

Reply:
@xferra this is for better understanding of handle func processes, because it returning array of objects [inside context], so, without looking into code of handle function, it is easy to understand principles of it.

Yes, you re sure, named and usage fields must be set to default values.